chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,65 @@
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    help = u'Creates AttachmentFinalization for the specified Attachment.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(u'-f', u'--file',
+                    dest=u'filename',
+                    help=u'define file path'),
+        make_option(u'--content_type',
+                    help=u'define content type of file',
+                    ),
+        make_option(u'--debug',
+                    default=u'',
+                    help=u'add debug message'
+                    ),
+        make_option(u'--force',
+                    action=u'store_true',
+                    help=u'overwrite an existing successful AttachmentFinalization'
+                    ),
+    )
+
+    def usage(self, subcommand=None):
+        return u'manage.py attachment_anonymization attachment [options] [file]'
+
+    def handle(self, *args, **options):
+        try:
+            filename = options[u'filename'] or args[1]
```

Comment:
V zadani pisem:
> file: Cesta k suboru, ktory sa nahraje ako AttachmentFinalization. Ak nie je uvedene, subor sa nacita zo stdin. Ak na stdin nie je nic, command failne.

Robi toto to, co je napisane v zadani?

Hlavne odkial si vycaroval to, ze mas 2 rozne file argumenty? Odkial si zobral `--file`, ked nic take v zadani nie je.

Reply:
@viliambalaz Neporozumeli sme sa. Pod "Ak nie je uvedene, subor sa nacita zo stdin" je myslene, ze **obsah suboru** sa nacita zo stdin, nie nazov suboru. Teda aby slo v skripte napisat nieco taketo:

```
nejaky-externy-tool | env/bin/python manage.py attachment_anonymization ...
```

Reply:
:ok: